### ISSUE TITLE
[Accessibility] Slides Page: Buttons do not have accessible name

### ISSUE THREAD

**Issue by nwanduka** [camicroscope/caMicroscope#969]
**Description**
The download and delete buttons do not have accessible names. When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.

**Screenshots**
![image](https://github.com/camicroscope/caMicroscope/assets/122698422/8499b624-223b-4f06-aaa1-a3583e609c6d)

**Proposed Fix**
Add an `aria-label` attribute to the button elements
